App/UI/GreenCertificate/ModalDetail/GreenCertificateVaccineDetail/GreenCertificateVaccineDetailView.swift

```diff
@@ -17,755 +17,797 @@ import Models
-        case .mRNA:
-            return Self.MRNA
-        case .antigen:
-            return Self.ANTIGEN
-        }
+  private static let COVID19 = "covid-19 vaccines"
+  private static let MRNA = "SARS-CoV-2 mRNA vaccine"
+  private static let ANTIGEN = "SARS-CoV-2 antigen vaccine"
+
+  case mRNA = "1119349007"
+  case antigen = "1119305005"
+  case covid19 = "J07BX03"
+
+  func getDescription() -> String {
+    switch self {
+    case .covid19:
+      return Self.COVID19
+    case .mRNA:
+      return Self.MRNA
+    case .antigen:
+      return Self.ANTIGEN
     }
+  }
 }
-public enum VaccineProducer: String {
 
-    case ORG_100001699 = "ORG-100001699"
-    case ORG_100030215 = "ORG-100030215"
-    case ORG_100001417 = "ORG-100001417"
-    case ORG_100031184 = "ORG-100031184"
-    case ORG_100006270 = "ORG-100006270"
-    case ORG_100013793 = "ORG-100013793"
-    case ORG_100020693 = "ORG-100020693"
-    case ORG_100010771 = "ORG-100010771"
-    case ORG_100024420 = "ORG-100024420"
-    case ORG_100032020 = "ORG-100032020"
-    case Gamaleya_Research_Institute = "Gamaleya-Research-Institute"
-    case Vector_Institute = "Vector-Institute"
-    case Sinovac_Biotech = "Sinovac-Biotech"
-    case Bharat_Biotech = "Bharat-Biotech"
-
-    func getDescription() -> String{
-        switch self {
-        case .ORG_100001699:
-            return "AstraZeneca AB"
-        case .ORG_100030215:
-            return "Biontech Manufacturing GmbH"
-        case .ORG_100001417:
-            return "Janssen-Cilag International"
-        case .ORG_100031184:
-            return "Moderna Biotech Spain S.L."
-        case .ORG_100006270:
-            return "Curevac AG"
-        case .ORG_100013793:
-            return "CanSino Biologics"
-        case .ORG_100020693:
-            return "China Sinopharm International Corp. - Beijing location"
-        case .ORG_100010771:
-            return "Sinopharm Weiqida Europe Pharmaceutical s.r.o. - Prague location"
-        case .ORG_100024420:
-            return "Sinopharm Zhijun (Shenzhen) Pharmaceutical Co. Ltd. - Shenzhen location"
-        case .ORG_100032020:
-            return "Novavax CZ AS"
-        case .Gamaleya_Research_Institute:
-            return "Gamaleya Research Institute"
-        case .Vector_Institute:
-            return "Vector Institute"
-        case .Sinovac_Biotech:
-            return "Sinovac Biotech"
-        case .Bharat_Biotech:
-            return "Bharat Biotech"
-        }
+public enum VaccineProducer: String {
+  case ORG_100001699 = "ORG-100001699"
+  case ORG_100030215 = "ORG-100030215"
+  case ORG_100001417 = "ORG-100001417"
+  case ORG_100031184 = "ORG-100031184"
+  case ORG_100006270 = "ORG-100006270"
+  case ORG_100013793 = "ORG-100013793"
+  case ORG_100020693 = "ORG-100020693"
+  case ORG_100010771 = "ORG-100010771"
+  case ORG_100024420 = "ORG-100024420"
+  case ORG_100032020 = "ORG-100032020"
+  case Gamaleya_Research_Institute = "Gamaleya-Research-Institute"
```

Comment:
<!--
  1 failure:  swiftlint (0.39.2...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: App/UI/GreenCertificate/ModalDetail/GreenCertificateVaccineDetail/GreenCertificateVaccineDetailView.swift;
  Line: 689;
-->
- :no_entry_sign: swiftlint (0.39.2): Enum element name should only contain alphanumeric characters: 'Gamaleya_Research_Institute' (identifier_name)